app/models/scoreboard.rb

```diff
@@ -56,7 +56,7 @@ def scored_schools(recent_boundary: 1.month.ago, academic_year: nil)
     if academic_year
       scored.joins(
         self.class.sanitize_sql_array(
-          ['LEFT JOIN observations ON observations.school_id = schools.id AND observations.at BETWEEN ? AND ?', academic_year.start_date, academic_year.end_date]
+          ['INNER JOIN observations ON observations.school_id = schools.id AND observations.at BETWEEN ? AND ?', academic_year.start_date, academic_year.end_date]
```

Comment:
This will have the effect of only showing schools on the scorboard if they have points. Is that intended?

Reply:
No, it's not intended - hmm, maybe it's worth us having a test to make sure that a school with no points appears - I'll need to remind myself why I changed it too ;)